src/app/shared/shared.module.ts

```diff
@@ -46,7 +47,8 @@ import { MaterialModule } from './material.module';
     IconsModule,
     MaterialModule,
     FormsModule,
-    ReactiveFormsModule
+    ReactiveFormsModule,
```

Comment:
remove this import

Reply:
When removing ReactiveFormsModule it's going to raise some issues in some components.

Reply:
oh sorry remove the comma

Reply:
There's a pipe module imported after that